src/mumble/MainWindow.ui

```diff
@@ -622,6 +622,17 @@ the channel's context menu.</string>
     <string>Shows the Mumble Developer Console, where Mumble's log output can be inspected.</string>
    </property>
   </action>
+  <action name="qaPositionalAudioViewer">
+   <property name="text">
+    <string>Positional &amp;Audio Viewer...</string>
+   </property>
+   <property name="toolTip">
+    <string>Show the Positional Audio Viewer</string>
+   </property>
+   <property name="whatsThis">
+    <string>Shows the Positional Audio Viewer, where current values can be inspected. Context and identity are also shown.</string>
```

Comment:
```suggestion
    <string>Shows the Positional Audio Viewer. In it the current positional information used for positional audio can be inspected. Context and identity are also shown.</string>
```

Reply:
What about omitting the first `positional`?

Reply:
Yeah sounds good :+1: